.env.example

```diff
@@ -4,6 +4,10 @@ NODE_ENV=production
 # Site URL (used for emails)
 SITE_URL=http://127.0.0.1
 
+# Websockets URL
+# Leave empty to use SITE_URL
```

Comment:
Leaving this empty will technically set the value to `''`

When it comes time to use the value:
```new WebSocket(`${wsUrl}/websocket/`);``` will be equal to:
```new WebSocket(`/websocket`);```

This will lead to the current URL of the page the user is on being taken, which will be a http or https route - resulting in a call to:
`https://saas.learninglocker.net/websocket` in the UI, for example

```suggestion
# Leave as comment to use SITE_URL
```